src/ompl/geometric/planners/bitstar/BITstar.h

```diff
@@ -375,118 +421,135 @@ namespace ompl
 
             /** \brief Retrieve the number of global-search edges that rewired the graph as a planner-progress property.
              * (numRewirings_) */
             std::string rewiringProgressProperty() const;
 
             /** \brief Retrieve the number of state collisions checks (i.e., calls to SpaceInformation::isValid(...)) as
-             * a planner-progress property. (from graphPtr_) */
+             * a planner-progress property. (From graphPtr_) */
             std::string stateCollisionCheckProgressProperty() const;
 
             /** \brief Retrieve the number of edge (or motion) collision checks (i.e., calls to
              * SpaceInformation::checkMotion(...)) as a planner-progress property. (numEdgeCollisionChecks_) */
             std::string edgeCollisionCheckProgressProperty() const;
 
             /** \brief Retrieve the number of nearest neighbour calls (i.e., NearestNeighbors<T>::nearestK(...) or
-             * NearestNeighbors<T>::nearestR(...)) as a planner-progress property. (from graphPtr_) */
+             * NearestNeighbors<T>::nearestR(...)) as a planner-progress property. (From graphPtr_) */
             std::string nearestNeighbourProgressProperty() const;
 
-            /** \brief Retrieve the total number of edges processed from the queue as a planner-progress property. (from
+            /** \brief Retrieve the total number of edges processed from the queue as a planner-progress property. (From
              * queuePtr_) */
             std::string edgesProcessedProgressProperty() const;
-            ///////////////////////////////////////////////////////////////////
 
-            ///////////////////////////////////////////////////////////////////
-            // Variables -- Make sure every one is configured in setup() and reset in clear():
-            /** \brief A helper for cost and heuristic calculations */
+            // ---
+            // Member variables (Make all are configured in setup() and reset in reset()).
+            // ---
+
+            /** \brief A helper for cost and heuristic calculations. */
             std::shared_ptr<CostHelper> costHelpPtr_{nullptr};
 
-            /** \brief The samples represented as an edge-implicit graph */
+            /** \brief The samples represented as an edge-implicit graph. */
             std::shared_ptr<ImplicitGraph> graphPtr_{nullptr};
 
             /** \brief The queue of vertices to expand and edges to process ordered on "f-value", i.e., estimated
              * solution cost. Remaining vertex queue "size" and edge queue size are accessible via
              * vertexQueueSizeProgressProperty and edgeQueueSizeProgressProperty, respectively. */
             std::shared_ptr<SearchQueue> queuePtr_{nullptr};
 
-            /** \brief The goal vertex of the current best solution */
+            /** \brief The inflation factor for the initial search. */
+            double initialInflationFactor_{1.0};
```

Comment:
I find the distinction between fooFactor_ and fooFactorParameter_ confusing. Can you think of a better naming convention to distinguish them? Also, I guess it's not possible to hide the ABIT* parameters and getter/setter methods from BIT*?

Reply:
I agree, this is confusing. I've changed the names and added more documentation in the ABIT* class description. I also made sure ABIT*'s parameters are only declared through `declareParam` when ABIT* is instantiated (is this what you meant?).